src/SuperstateToken.sol

```diff
@@ -25,9 +26,6 @@ abstract contract SuperstateToken is ERC20Upgradeable, IERC7246, PausableUpgrade
     bytes32 internal constant DOMAIN_TYPEHASH =
         keccak256("EIP712Domain(string name,string version,uint256 chainId,address verifyingContract)");
 
-    /// @notice Admin address with exclusive privileges for minting and burning
```

Comment:
are we screwing with storage by deleting this? wont the proxy expect the storage layout to be the same?

Reply:
good catch, I think that's correct.

Reply:
IIRC we should keep, we can rename to make it clear that it isnt to be used anymore